doc/api/esm.md

```diff
@@ -659,26 +659,29 @@ The resolve hook returns the resolved file URL and module format for a
 given module specifier and parent file URL:
 
 ```js
-import { URL, pathToFileURL } from 'url';
-const baseURL = pathToFileURL(process.cwd()).href;
+import { pathToFileURL } from 'url';
 
 /**
  * @param {string} specifier
  * @param {string} parentModuleURL
- * @param {function} defaultResolver
+ * @param {Function} defaultResolver
  */
 export async function resolve(specifier,
-                              parentModuleURL = baseURL,
+                              parentModuleURL = undefined,
```

Comment:
Do we care about preserving the function length of this example of using an experimental API? And if that value is always provided when calling, providing a default seems misleading?

Reply:
@jkrems I was hoping that we could eventually use a templating engine to import example fixtures like this directly from their JS sources into their MD destinations. This would guarantee that the code samples are tested for accuracy and never desync. Just a thought to consider though, please carry on.

Reply:
If it can be omitted, it should not be included in the length. Making it have a default is the mechanism to do this.